src/index.html

```diff
@@ -2,17 +2,90 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
+
     <meta
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
+
     <title>Product cards</title>
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
+
   <body>
-    <h1>Product cards</h1>
+    <section
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__image">
+        <img
+          class="card__image-photo"
+          src="images/imac.jpeg"
+          alt="product photo"
+        />
+      </div>
+
+      <h2 class="card__model">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <div class="card__product-code">
+        <p class="card__product-code-style">Product code: 195434</p>
+      </div>
+
+      <div class="card__grade">
+        <div class="card__grade-stars">
+          <ul class="card__grade-stars-list">
+            <li class="card__grade-stars-list-1"></li>
+
+            <li class="card__grade-stars-list-2"></li>
+
+            <li class="card__grade-stars-list-3"></li>
+
+            <li class="card__grade-stars-list-4"></li>
+
+            <li class="card__grade-stars-list-5"></li>
+          </ul>
+        </div>
+
+        <div class="card__grade-reviews">Reviews: 5</div>
+      </div>
+
+      <div class="card__price">
+        <p class="card__price-style">Price:</p>
```

Comment:
Similar to the previous comment, the `p` tag with class `card__price-style` should follow the BEM naming convention. It could be named `card__price-label` to indicate its purpose more clearly.